Makefile

```diff
@@ -50,9 +50,10 @@ test-slow-matrix: og
 	-turnt --save --env matrix_oracle test/*.og
 	turnt --diff -v --env matrix_test test/*.gfa
 
-test-slow-validate: og
-	-turnt --save --env validate_oracle test/*.gfa
-	turnt --diff -v --env validate_test test/*.gfa
+test-slow-validate: fetch
+	-turnt --save --env validate_setup/*.gfa
```

Comment:
I need to learn more about turnt, but I can't get this little chunk to run. What I want is:
- Run `perturb` to generate, for each file.gfa, a new temp.file.gfa. This temp.file.gfa will be missing a few links, thus giving validate something to work with.
- Run `odgi validate` on temp.file.gfa, saving the result of this as file.out
- Run slow-odgi's validate on temp.file.gfa, comparing against file.out

However, it seems to me that the test-slow-validate stanza is not running at all right now.

Reply:
I've done up a quick band-aid solution here: [Temp fix to generate interesting cases for validate](https://github.com/cucapra/pollen/pull/29/commits/6a909ba88090ca9eddaa7c3f8ed1642513090eb3) but I'd like to actually get it right using turnt